randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1455,21 +1470,29 @@ Extra - map_name: rm_a5b05
   > Tunnel to Spiky Spider Trial Reward Room
       Any of the following:
           All of the following:
-              Morph Ball and Mid-Air Morph (Advanced)
+              # Suitless Method
+              Morph Ball
+              Any of the following:
+                  Mid-Air Morph (Advanced)
+                  Power Grip and Movement (Intermediate)
               Any of the following:
                   All of the following:
                       Walljump (Intermediate)
-                      Power Grip or Mid-Air Morph (Expert)
+                      Power Grip or Mid-Air Morph (Advanced)
                   All of the following:
                       # IBJ
                       Infinite Bomb Jumping (Advanced) and Can Use Bombs
-          Gravity Suit and Can Use Spider Ball
+          All of the following:
+              # Intended Method
+              Gravity Suit and Can Use Spider Ball
 
 > Tunnel to Spiky Spider Trial Reward Room; Heals? False
   * Layers: default
   * Morph Ball Tunnel to Spiky Spider Trial Reward Room/Tunnel to Spiky Spider Trial
   > Dock to Serris Arena Access
-      Trivial
+      All of the following:
+          Morph Ball
+          Bombs or Power Grip or Spring Ball or Mid-Air Morph (Beginner)
```

Comment:
Make this "tunnel climb or mid air morph "please

Reply:
tunnel climb implies the option for spider ball which needs gravity

Reply:
ah crap, you're right.